app/controllers/application_controller/ci_processing.rb

```diff
@@ -1868,6 +1868,7 @@ def process_cloud_object_storage_buttons(pressed)
     task = pressed.sub("#{klass.name.underscore.to_sym}_", "")
 
     return tag(klass) if task == "tag"
+    return javascript_redirect(:controller => klass.name.underscore.to_sym, :action => "new") if task == "new"
 
```

Comment:
I hope not; I think that creating a new bucket from the list of provider's buckets should be supported.